### PR TITLE
chore(Component Generator): Use Named Exports for Components

### PR DESCRIPTION
**Pull request by deniaz**

### CODE REVIEW

File: scripts/templates/component.ts [hunk 1]
```diff
@@ -4,15 +4,15 @@ import ComponentFileBuilderResponse from "./ComponentFileBuilderResponse";
...
+import { FC } from "react";
 
-export interface ${componentName}Props {
+export type ${componentName}Props = {
     foo: string;
-}
+};
 
-export default function ${componentName}({ foo }: ${componentName}Props): ReactElement<${componentName}Props> {
+export const ${componentName}: FC<${componentName}Props> = ({ foo }) => {
```
Comment (SamuelAlev) [inline scripts/templates/component.ts:13]:
```suggestion
export const ${componentName}: FC<${componentName}Props> = ({ foo }: ${componentName}Props) => {
```

File: scripts/templates/component.stories.ts [hunk 2]
```diff
@@ -6,17 +6,22 @@ export default (componentName: string, componentType: string): ComponentFileBuil
...
+export const Default = Template.bind({});
+
+Default.args = {
     args: {
         foo: "bar",
     },
-} as Meta<${componentName}Props>;
-
-export const ${componentName}: Story<${componentName}Props> = (args) => <${componentName}Component {...args} />;
+};
```
Comment (SamuelAlev) [inline scripts/templates/component.stories.ts:24]: One `args` too much :)

File: .eslintrc.js [hunk 3]
```diff
@@ -6,4 +6,7 @@ module.exports = {
         },
     },
     plugins: ["jsx-a11y"],
+    rules: {
+        "react/prop-types": "off",
+    },
```
Comment (SamuelAlev) [inline .eslintrc.js:11]: Can be removed

Comment (dwirz) [reply .eslintrc.js]: works 👍  removed the rule ee61723

File: scripts/templates/component.ts (hunk 1 above)
Comment (dwirz) [reply scripts/templates/component.ts:14]: no need since the `FC` type ensures that props have the correct typing

**Pull request merged**